Startup/startup_ch32x035.S

```diff
@@ -0,0 +1,226 @@
+    .weak   EXTI25_16_IRQHandler       /* EXTI Line 25..16 */
+    .weak   USART3_IRQHandler          /* USART3 */
+    .weak   USART4_IRQHandler          /* USART4 */
+    .weak   DMA1_Channel8_IRQHandler   /* DMA1 Channel8 */
+    .weak   USBFS_IRQHandler           /* USBFS Break */
+    .weak   USBFSWakeUp_IRQHandler     /* USBFS Wake up from suspend */
+    .weak   PIOC_IRQHandler            /* PIOC */
+    .weak   OPA_IRQHandler             /* OPA */
+    .weak   USBPD_IRQHandler           /* USBPD */
+    .weak   USBPDWakeUp_IRQHandler     /* USBPD Wake up */
+    .weak   TIM2_CC_IRQHandler         /* TIM2 Capture Compare */
+    .weak   TIM2_TRG_COM_IRQHandler    /* TIM2 Trigger and Commutation */
+    .weak   TIM2_BRK_IRQHandler        /* TIM2 Break */
+    .weak   TIM3_IRQHandler            /* TIM3 */
+
+NMI_Handler:  1:  j 1b
+HardFault_Handler:  1:  j 1b
+Ecall_M_Mode_Handler:  1:  j 1b
+Ecall_U_Mode_Handler:  1:  j 1b
+Break_Point_Handler:  1:  j 1b
+SysTick_Handler:  1:  j 1b
+SW_Handler:  1:  j 1b
+WWDG_IRQHandler:  1:  j 1b
+PVD_IRQHandler:  1:  j 1b
+FLASH_IRQHandler:  1:  j 1b
+EXTI7_0_IRQHandler:  1:  j 1b
+AWU_IRQHandler:  1:  j 1b
+DMA1_Channel1_IRQHandler:  1:  j 1b
+DMA1_Channel2_IRQHandler:  1:  j 1b
+DMA1_Channel3_IRQHandler:  1:  j 1b
+DMA1_Channel4_IRQHandler:  1:  j 1b
+DMA1_Channel5_IRQHandler:  1:  j 1b
+DMA1_Channel6_IRQHandler:  1:  j 1b
+DMA1_Channel7_IRQHandler:  1:  j 1b
+ADC1_IRQHandler:  1:  j 1b
+I2C1_EV_IRQHandler:  1:  j 1b
+I2C1_ER_IRQHandler:  1:  j 1b
+USART1_IRQHandler:  1:  j 1b
+SPI1_IRQHandler:  1:  j 1b
+TIM1_BRK_IRQHandler:  1:  j 1b
+TIM1_UP_IRQHandler:  1:  j 1b
+TIM1_TRG_COM_IRQHandler:  1:  j 1b
+TIM1_CC_IRQHandler:  1:  j 1b
+TIM2_UP_IRQHandler:  1:  j 1b
+USART2_IRQHandler:  1:  j 1b
+EXTI15_8_IRQHandler:  1:  j 1b
+EXTI25_16_IRQHandler:  1:  j 1b
+USART3_IRQHandler:  1:  j 1b
+USART4_IRQHandler:  1:  j 1b
+DMA1_Channel8_IRQHandler:  1:  j 1b
+USBFS_IRQHandler:  1:  j 1b
+USBFSWakeUp_IRQHandler:  1:  j 1b
+PIOC_IRQHandler:  1:  j 1b
+OPA_IRQHandler:  1:  j 1b
+USBPD_IRQHandler:  1:  j 1b
+USBPDWakeUp_IRQHandler:  1:  j 1b
+TIM2_CC_IRQHandler:  1:  j 1b
+TIM2_TRG_COM_IRQHandler:  1:  j 1b
+TIM2_BRK_IRQHandler:  1:  j 1b
+TIM3_IRQHandler:  1:  j 1b
+
+	.section	.text.handle_reset,"ax",@progbits
+	.weak	handle_reset
+	.align	1
+handle_reset:
+.option push 
+.option	norelax 
+	la gp, __global_pointer$
+.option	pop 
+1:
+	la sp, _eusrstack 
+2:
+	/* Load data section from flash to RAM */
+	la a0, _data_lma
+	la a1, _data_vma
+	la a2, _edata
+	bgeu a1, a2, 2f
+1:
+	lw t0, (a0)
+	sw t0, (a1)
+	addi a0, a0, 4
+	addi a1, a1, 4
+	bltu a1, a2, 1b
+2:
+	/* Clear bss section */
+	la a0, _sbss
+	la a1, _ebss
+	bgeu a0, a1, 2f
+1:
+	sw zero, (a0)
+	addi a0, a0, 4
+	bltu a0, a1, 1b
+2:
+    li t0, 0x1f
+    csrw 0xbc0, t0
+
+    /* Enable nested and hardware stack */
+	li t0, 0x3
+	csrw 0x804, t0
+
+    /* Enable interrupt */
+   	li t0, 0x88           
+   	csrs mstatus, t0
```

Comment:
This 0x88 will value for the mstatus register will work for normal `noneos-sdk` firmwares, but later we'll need to change it like [the other ones](https://github.com/Community-PIO-CH32V/framework-wch-noneos-sdk/blob/2cafe7893404eb413d939eef6ae6e2e77063e7af/Startup/startup_ch32v10x.S#L222-L250) to support running RTOS frameworks (FreeRTOS, Harmony LiteOS, RT-Thread, TencentOS,..) on it, if it supports it.